src/mixins/nft.js

```diff
@@ -611,7 +611,7 @@ export default {
       this.lazyGetUserInfoByAddresses(this.iscnOwner);
     },
     async updateNFTClassAggregatedInfo() {
```

Comment:
changing the name of this function to `lazyFetchNFTClassAggregatedInfo` might be bette, avoid future confusion about how to actually `update` an already fetched info